app/views/vet-visit-date.njk

```diff
@@ -29,12 +40,32 @@
             }
           },
           hint: {
-            html: "This is the date of the last visit to the farm for the review<br>For example, 27 3 2022"
+            html: "This is the date when the vet last visited the farm to carry out the review.<br>For example, 27 3 2022"
           },
-          errorMessage: errorMessage,
+          errorMessage: errorList.visitDateError,
           items: items
         }) }}
 
+        <h2 class="govuk-heading-m">When was endemic disease or condition testing carried out?</h2>
+
+        {{ govukRadios({
+          name: "testing-date",
```

Comment:
Haven't done a full review as I'm on my phone but there is a feature toggle for the date of testing that needs to be used.